certified-connectors/Impexium/apiDefinition.swagger.json

```diff
@@ -930,93 +396,80 @@
             "default": "application/json",
             "x-ms-visibility": "internal"
           }
-        ]
+        ],
+        "summary": "Get Organization relationships",
```

Comment:
Please change to `Get Organization's Relationships`, to match the format of the Individual's equivalent action